PadoruLogger/Debug.cs

```diff
@@ -306,7 +243,7 @@ private static void AutoConfigure()
 
             AddOutput(defaultOutput);
 
-            LogWarning($"Tried to use Padoru.Diagnostics.Debug without configuring it first. Logger auto-configured itself with default options.");
+            LogWarning($"Tried to use Padoru.Diagnostics.Debug without configuring it first. Logger auto-configured itself with default options.", "LogInternal");
```

Comment:
This channel should be a constant and called something more informative like `PadoruDebug`